packages/javascript/bh-shared-ui/src/views/PrivilegeZones/History/columns.tsx

```diff
@@ -28,9 +28,9 @@ const actionTranslate: Record<string, string> = {
     DeleteTag: 'Delete Tag',
     AnalysisEnabledTag: 'Analysis Enabled Tag',
     AnalysisDisabledTag: 'Analysis Disabled Tag',
-    CreateSelector: 'Create Selector',
-    UpdateSelector: 'Update Selector',
-    DeleteSelector: 'Delete Selector',
+    CreateRule: 'Create Rule',
+    UpdateRule: 'Update Rule',
+    DeleteRule: 'Delete Rule',
```

Comment:
blocking suggestion: These keys are not updated in the backend, only the value should be updated. Though that would also break searching by the action and relegate that to only the filter